example/lib/main.dart

```diff
@@ -24,10 +24,12 @@ class _MyAppState extends State<MyApp> {
         visualDensity: VisualDensity.adaptivePlatformDensity,
       ),
       home: Scaffold(
+        backgroundColor: Colors.grey,
```

Comment:
can we showcase the adjustable background color?

Reply:
This backround color just shows the background color of the page selected